diff --git a/gpt4all-chat/CMakeLists.txt b/gpt4all-chat/CMakeLists.txt
index e19b1c4e..efbf1a19 100644
--- a/gpt4all-chat/CMakeLists.txt
+++ b/gpt4all-chat/CMakeLists.txt
@@ -94,7 +94,7 @@ endif()
 # Embedding model
 set(LOCAL_EMBEDDING_MODEL "nomic-embed-text-v1.5.f16.gguf")
 set(LOCAL_EMBEDDING_MODEL_MD5 "a5401e7f7e46ed9fcaed5b60a281d547")
-set(LOCAL_EMBEDDING_MODEL_PATH "${CMAKE_BINARY_DIR}/resources/${LOCAL_EMBEDDING_MODEL}")
+set(LOCAL_EMBEDDING_MODEL_PATH "${CMAKE_CURRENT_SOURCE_DIR}/resources/${LOCAL_EMBEDDING_MODEL}")
 set(LOCAL_EMBEDDING_MODEL_URL "https://gpt4all.io/models/gguf/${LOCAL_EMBEDDING_MODEL}")
 message(STATUS "Downloading embedding model from ${LOCAL_EMBEDDING_MODEL_URL} ...")
 file(DOWNLOAD
